google/install-chrome/README.md

```diff
@@ -77,5 +77,120 @@ The following output values are available:
 - `${{ tasks.chrome.values.chrome-directory }}`
 - `${{ tasks.chrome.values.chromedriver-binary }}`
 - `${{ tasks.chrome.values.chromedriver-directory }}`
+- `${{ tasks.chrome.values.start-virtual-display-binary }} `
+- `${{ tasks.chrome.values.is-virtual-display-running-binary }} `
 
-The `chromedriver.*` values are only available when chromedriver is installed.
+The `chromedriver-*` values are only available when chromedriver is installed.
+
+# Using headless Chrome
+
+By default, `google/install-chrome` supports tools that interact with Chrome in headless mode. Here's an example with Selenium and Ruby:
+
+```yml
+- key: chrome
+  call: google/install-chrome 1.1.0
+  with:
+    chrome-version: stable
+    install-chromedriver: true
+
+- key: ruby
+  call: mint/install-ruby 1.1.0
+  with:
+    ruby-version: 3.3.4
+
+- key: selenium-example
+  use: [chrome, ruby]
+  run: |
+    cat << EOF > Gemfile
+    source "https://rubygems.org"
+
+    gem "selenium-webdriver", "~> 4.24"
+    EOF
+
+    cat << EOF > selenium.rb
+    require "selenium-webdriver"
+
+    Selenium::WebDriver.logger.level = :debug
+    Selenium::WebDriver.logger.output = 'selenium.log'
+
+    options = Selenium::WebDriver::Options.chrome(args: ['--headless=new'])
+    driver = Selenium::WebDriver.for(:chrome, options:)
+    driver.navigate.to "http://google.com"
+
+    element = driver.find_element(name: 'q')
+    element.send_keys "Hello WebDriver!"
+    element.submit
+
+    puts driver.title
+
+    driver.quit
+    EOF
+
+    bundle install
+
+    ruby selenium.rb | grep "Hello WebDriver! - Google Search"
+    cat selenium.log
+```
+
+# Using headed Chrome
+
+You can also use tools that interact with headed Chrome. To do so, you'll need to add a background process to the task that's using Chrome. Here's an example with Selenium and Ruby:
+
+```yml
+- key: chrome
+  call: google/install-chrome 1.1.0
+  with:
+    chrome-version: stable
+    install-chromedriver: true
+
+- key: ruby
+  call: mint/install-ruby 1.1.0
+  with:
+    ruby-version: 3.3.4
+
+- key: selenium-example
+  use: [chrome, ruby]
+  background-processes:
+    - key: chrome-virtual-display
+      run: start-chrome-virtual-display
+      ready-check: is-chrome-virtual-display-running
```

Comment:
Do we want to suggest folks use the version in the path or the specific value from the task outputs?

```suggestion
      run: ${{ tasks.chrome.values.start-virtual-display-binary }}
      ready-check: ${{ tasks.chrome.values.is-virtual-display-running-binary }}
```

Reply:
I think we want to suggest the one in the path.

The outputs are really only there in case you need multiple chromes. Honestly even in that case it's maybe questionable since you know the chrome and chromedriver dir - maybe we should just remove the binaries from the values?

Reply:
I agree - using the version in PATH is nicer, and if we're committing to the name within the chrome directory output path, removing the binary paths from the output would remove that point of confusion for me. (Mostly, if the outputs are there, I'd feel obligated to use them instead of treating the executable name as part of the interface for this leaf.)